tests/fpga/hbm_multiple_interface_test.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019-2021 ETH Zurich and the DaCe authors. All rights reserved.
+
+from dace.transformation.dataflow.map_unroll import MapUnroll
+from dace import dtypes, subsets
+import dace
+from dace import memlet
+from dace.fpga_testing import xilinx_test
+import numpy as np
+from dace.sdfg import SDFG
+from dace.transformation.interstate import InlineSDFG
+
+# Checks multiple interfaces attached to the same HBM-bank.
+# TODO: update
+
+
+@xilinx_test(assert_ii_1=False)
+def test_3_interface_to_2_banks():
+    sdfg = SDFG("test_4_interface_to_2_banks")
+    state = sdfg.add_state()
+
+    _, desc_a = sdfg.add_array("a", [2, 2], dace.int32)
+    desc_a.location["memorytype"] = "HBM"
+    desc_a.location["bank"] = "0:2"
+    acc_read1 = state.add_read("a")
+    acc_write1 = state.add_write("a")
+
+    t1 = state.add_tasklet("r1", set(["_x1", "_x2"]), set(["_y1"]),
+                           "_y1 = _x1 + _x2")
+
+    m1_in, m1_out = state.add_map("m", {"k": "0:2"},
+                                  dtypes.ScheduleType.Unrolled)
+
+    state.add_memlet_path(acc_read1,
+                          m1_in,
+                          t1,
+                          memlet=memlet.Memlet("a[0, 0]"),
+                          dst_conn="_x1")
+    state.add_memlet_path(acc_read1,
+                          m1_in,
+                          t1,
+                          memlet=memlet.Memlet("a[1, 0]"),
+                          dst_conn="_x2")
+    state.add_memlet_path(t1,
+                          m1_out,
+                          acc_write1,
+                          memlet=memlet.Memlet("a[0, 1]"),
+                          src_conn="_y1")
+
+    sdfg.apply_fpga_transformations()
+    sdfg.apply_transformations(InlineSDFG)
+    sdfg.apply_transformations(MapUnroll)
```

Comment:
You should assert that the right number of transformations are actually applied here, i.e.:
```python
assert sdfg.apply_transformations(InlineSDFG) == 1
```